src/components/BookItem/BookItem.css

```diff
@@ -54,7 +54,7 @@ ul {
 
 .progress {
   display: flex;
```

Comment:
- [ ] It would be better if we implement the CSS DRY Principles, here is a way to implement it since we have repeating rules like for example; `display: flex`, `justify-content: space-between;`, etc